tests/integration/longhorn/longhorn_int_test.go

```diff
@@ -0,0 +1,154 @@
+package longhorn
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+
+	testutil "github.com/k3s-io/k3s/tests/integration"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var server *testutil.K3sServer
+var serverArgs = []string{"--cluster-init"}
+var testLock int
+
+var _ = BeforeSuite(func() {
+	if _, err := exec.LookPath("iscsiadm"); err != nil {
+		Skip("Test needs open-iscsi to be installed")
```

Comment:
What setups this up?

Reply:
Please open a discussion if you have questions. Year old PRs are not the correct place.